src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/request/PrisonUserSearchRequest.kt

```diff
@@ -1,15 +1,15 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.request
 
 import io.swagger.v3.oas.annotations.media.Schema
-import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.NotEmpty
 
 @Schema(description = "Request object for searching for offenders within a set of teams attached to a staff member")
 data class PrisonUserSearchRequest(
 
   @Schema(description = "The query the user wishes to search for (e.g. CRN, name, NOMIS ID)", example = "Joe Bloggs")
   val query: String,
 
-  @Schema(description = "The prison caseloads of the prison staff member", example = "BAI")
-  @field:NotNull
-  val prisonCaseload: String,
+  @Schema(description = "The prison caseloads of the prison staff member", example = "[BAI]")
+  @field:NotEmpty
+  val prisonCaseload: List<String>,
```

Comment:
Guess this should be`prisonCaseLoads`, and you could make this a `Set` so you don't need to transform lower down?